app/Models/Multiplayer/Room.php

```diff
@@ -338,6 +356,42 @@ public function completePlay(Score $score, array $params)
         });
     }
 
+    public function findAndSetCurrentPlaylistItem(): PlaylistItem
```

Comment:
this will error if a `Room` has no playlist items (whose expected behaviour is currently being worked out 👀 )